### ISSUE TITLE
Comune di Aosta - Richiesta certificazione massiva

### ISSUE THREAD

**Issue by RBiasi** [italia/anpr#1132]
Buongiorno,

vorremmo sapere se è previsto WS per richiedere la certificazione ANPR in modo massivo.

Ovvero dato un range specifico, ad esempio l'indirizzo di residenza, produrre gli stai di famiglia di tutti i cittadini in maniera massiva.

La presente richiesta sorge al fine di evadere richieste ad esempio provenienti dalle forze dell'ordine.

Roberta Biasi
Enterprise Solution Consultant
Solutions Operations & Capabilities Development-Nord & Centro
Dedagroup Public Sector & Utilities

**Comment by FilippoPapi** [comment]
No, non è previsto. il gestionale volendo potrebbe richiamare il servizio in modo ricorsivo.
